
Turn 2
================================================================
error: failed to decode bot message: "T" (at position 1)
Last input received was:
Traceback (most recent call last):
  File "MyBot.py", 
